source/tomopy/prep/stripe.py

```diff
@@ -939,3 +941,156 @@ def _remove_stripe_based_interpolation(tomo, snr, size, drop_ratio, norm):
         sino = tomo[:, m, :]
         sino = _rs_interpolation(sino, snr, size, drop_ratio, norm)
         tomo[:, m, :] = sino
+
+
+def stripes_detect3d(arr, vert_filter_size_perc=5, radius_size=3, ncore=None):
+    """
+    Apply a stripes detection method to empasize their edges in a 3D array.
+    The input must be normalized projection data in range [0,1] and given in
+    the following axis orientation [angles, detY(depth), detX (horizontal)]. With 
+    this orientation, the stripes are the vertical features. The method works with
+    full and partial stripes of constant ot varying intensity. 
+
+    .. versionadded:: 1.13
```

Comment:
```suggestion
    .. versionadded:: 1.14
```
We use semantic versioning. Since 1.13 is already released, this feature will first appear in 1.14.

Reply:
fixed in 4974eee